packages/core/bootstrap/src/lib/cache/redis/redis.ts

```diff
@@ -43,32 +59,9 @@ export const redactOptions = (opts: RedisOptions) => {
   return opts
 }
 
-const retryStrategy: RetryStrategy = (options) => {
-  metrics.redis_retries_count.inc()
-  logger.warn('Redis retry strategy activated.', options)
-  if (options.error && options.error.code === 'ECONNREFUSED') {
-    // End reconnecting on a specific error and flush all commands with
-    // a individual error
-    logger.warn('Connection refused.', options)
-    return new Error('The server refused the connection')
-  }
-  if (options.total_retry_time > 1000 * 60 * 60) {
-    // End reconnecting after a specific timeout and flush all commands
-    // with a individual error
-    logger.warn('Redis retry strategy exhausted.', options)
-    return new Error('Retry time exhausted')
-  }
-  if (options.attempt > 10) {
-    logger.warn(`Redis retry attempt #${options.attempt}`, options)
-    return undefined
-  }
```

Comment:
Looks like all of these cases would stop the retry strategy previously, but is not included in your changes. Should we include these as well?

Reply:
Oooh I read your PR description now.
> Simplify the new reconnectStrategy from retryStrategy since the new version only allows the number of retries as an input argument, without any other data

Is there a way to limit the number of commands in the queue btw?

Reply:
If you have any ideas to keep some of this logic, lmk. I certainly don't want to get rid of it if there is an alternative.

Reply:
It does look like we can limit the queue though. I might dig into the `createClient` options interface and see what other configuration settings there are:
```
export interface RedisClientOptions<M extends RedisModules, S extends RedisScripts> extends RedisPlugins<M, S> {
    url?: string;
    socket?: RedisSocketOptions;
    username?: string;
    password?: string;
    database?: number;
    commandsQueueMaxLength?: number;
    readonly?: boolean;
    legacyMode?: boolean;
    isolationPoolOptions?: PoolOptions;
}

Reply:
If this is correct, there are a lot of options for the `isolationPoolOptions` as well:
https://github.com/coopernurse/node-pool/blob/1c5cb79dcbea27c4b1839bd75bfc41274adb8b94/lib/PoolOptions.js